deployment-config.json

```diff
@@ -0,0 +1,56 @@
+{
+    "biomage-org": {
+        "domain_name": "Biomage",
+        "environment_names": [
+            "alabs",
+            "Biomage",
+            "trv"
+        ],
+        "docker_image_account": {
+            "id": 242905224710,
+            "region": "eu-west-1"
+        },
+        "support_email": "engineering@biomage.net",
+        "reply_email": "hello@biomage.net",
+        "deployment_environments": {
+            "trv": [
+                "production"
+            ],
+            "alabs": [
+                "production"
+            ],
+            "Biomage": [
+                "production",
+                "staging"
+            ]
+        },
+        "excluded_services": [
+            "cf/rds-agent-security-group.yaml",
+            "cf/rds-subnet-security-group.yaml"
+        ]
+    },
+    "hms-dbmi-cellenics": {
+        "domain_name": "HMS",
+        "environment_names": [
+            "HMS"
+        ],
+        "docker_image_account": {
+            "id": 160782110667,
+            "region": "us-east-1"
+        },
+        "support_email": "alex_pickering@hms.harvard.edu",
+        "reply_email": "alex_pickering@hms.harvard.edu",
+        "deployment_environments": {
+            "HMS": [
+                "staging",
+                "production"
+            ]
+        },
+        "excluded_services": [
+            "cf/ses.yaml",
+            "cf/doit-cloudhealth-role.yaml",
+            "cf/rds-agent-security-group.yaml",
+            "cf/rds-subnet-security-group.yaml"
+        ]
```

Comment:
can **datadog** be excluded as well? It doesn't seem to fit quite so nicely into a single cf file.

HMS doesn't use datadog for our deployment -- ideally it would be nice to not deploy all the stuff related to it

Reply:
These excluded services concern the deploy-changed-cf script. Datadog services are already excluded - we have a check, for the existence of the DATADOG_API_KEY and if  it doesn't exist it skips the step "Setup Datadog cluster agent"

Reply:
Ah yes, I see that this was added (or maybe fixed) two weeks ago. I will look into removing the datadog-agent deployment from the HMS instances